lib/importers/article_importer.rb

```diff
@@ -31,7 +37,9 @@ def self.import_articles_by_title(titles)
       results.each do |_id, page_data|
         next if page_data['missing']
         articles << Article.new(id: page_data['pageid'].to_i,
+                                mw_page_id: page_data['pageid'].to_i,
```

Comment:
Refers to page_data more than self (maybe move it to another class?) (FeatureEnvy)